modules/web/hosting-environment/main.bicep

```diff
@@ -154,7 +152,7 @@ resource appServiceEnvironment 'Microsoft.Web/hostingEnvironments@2022-03-01' =
   kind: kind
   location: location
   tags: tags
-  identity: identity
+  identity: identity!
```

Comment:
I guess that's a warning? But using `!` as a means to claim it will never be null seems wrong.

Reply:
As discussed, changed it to any() in the definition of the "identity" variable, which also suppresses the warning.